content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md

```diff
@@ -73,6 +73,16 @@ Ask a maintainer from the PrestaShop Company with access to the Translation Tool
 It is usually only done once per release as well.
 {{% /notice %}}
 
+1. [Use the command to extract wordings from the TranslationTool repository](https://github.com/PrestaShopCorp/TranslationTool/actions/workflows/create-default-catalog-pr.yml). This command will automatically generates a Pull Request on the PrestaShop/PrestaShop repository (author should be jarvis). It is important to note that this PR must be reviewed by a member of the content team.
+
```

Comment:
Can you please add a line below that mentions that https://github.com/PrestaShopCorp/TranslationTool is PRIVATE so people need to be granted access to it to be able to use it 😉 ?

Reply:
Done